### PR TITLE
Change default text

### LINKED ISSUE #3: Change default SDG text and images.
Addresses Defra-Data-Science-Centre-of-Excellence/OIF-Dashboard-Site#27

### PR DESCRIPTION
**Pull request by EFT-Defra**
Closes #3 by editing files in /translations/en

### CODE REVIEW

File: translations/en/general.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+indicator: Indicator
+indicators: Indicators
+goal: Goal
+goals: Goals
+sdg: Outcome Indicator Framework
```
Comment (EFT-Defra) [inline translations/en/general.yml:5]: Change SDG to OIF

File: translations/en/general.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+target: Target
+targets: Targets
+twitter: Twitter
+facebook: Facebook
+built_using_open_sdg: Built using Open SDG
+page: Page
+pages: Pages
+targets_and_indicators: Targets and indicators
+label_warning: "Warning:"
+site_title: Outcome Indicator Framework for the 25 Year Environment Plan
```
Comment (EFT-Defra) [inline translations/en/general.yml:19]: Change to OIF title

File: translations/en/general.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+sdg: Outcome Indicator Framework
+home: Home
+loading: Loading search results
+opens_new_window: opens in a new window
+hide: hide
+target: Target
+targets: Targets
+twitter: Twitter
+facebook: Facebook
+built_using_open_sdg: Built using Open SDG
```
Comment (EFT-Defra) [inline translations/en/general.yml:14]: I though I should leave this, as it is built using Open SDG

File: translations/en/header.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+skip_link: Skip to main content
+alpha: Alpha
+disclaimer: This is a development website. We welcome your
+  <a href="mailto:%email_address">feedback</a>.
+tag_line: Improving the environment within a generation
```
Comment (EFT-Defra) [inline translations/en/header.yml:5]: I lifted this from [here](https://www.gov.uk/government/publications/25-year-environment-plan)

File: translations/en/frontpage.yml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+instructions: Click on each goal, or %before_link search, %after_link
+  for %name statistics for Sustainable Development Goal global
+  indicators.
+heading: "%name data for Sustainable Development Goal indicators"
+download_all: Download all data
+zip_file: ZIP file
+intro_title: Outcome Indicator Framework for the 25 Year Environment Plan
+intro_body: This is government’s annual account of how it has implemented its commitments for protecting and improving the environment. It is our second account of progress since we published the <a href="https://www.gov.uk/government/publications/25-year-environment-plan">25 Year Environment Plan</a>, our plan for improving the environment within a generation.
```
Comment (EFT-Defra) [inline translations/en/frontpage.yml:8]: Copied over from [here](https://github.com/Defra-Data-Science-Centre-of-Excellence/OIF-Dashboard-Site/blob/develop/_config.yml)

**Review by TimAshelford — APPROVED** [review]

**Pull request merged**